setup-build.py

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+
+import os
+import platform
+
+dist = platform.dist()[0]
+common_tools = "cmake flex bison"
+if dist == 'Ubuntu':
+	os.system('apt-get install -y build-essential {}'.format(common_tools))
```

Comment:
If someone decide to build from source it's up to them to have a decent developing environment